src/main/webapp/app/overview/course-exams/course-exams.component.ts

```diff
@@ -60,4 +109,40 @@ export class CourseExamsComponent implements OnInit, OnDestroy {
     isVisible(exam: Exam): boolean {
         return exam.visibleDate ? dayjs(exam.visibleDate).isBefore(this.serverDateService.now()) : false;
     }
+
+    /**
+     * Returns if the given exam is a TestExam
+     * @param exam the exam to be checked
+     */
+    isTestExam(exam: Exam): boolean {
+        return exam.testExam ? exam.testExam : false;
+    }
```

Comment:
Is this necessary? Seems like ``Exam#testExam`` is a ``boolean`` already?
If it's required, use
```suggestion
    /**
     * Returns if the given exam is a TestExam
     * @param exam the exam to be checked
     * @returns true if this exam is a TextExam; false otherwise
     */
    isTestExam(exam: Exam): boolean {
        return !!exam.testExam;
    }
```
although you could also just use that line everywhere you need it and drop this method, that's probably even shorter

Reply:
I originally used this method to pass the Boolean into an angular component. 
But this is no longer needed, and I can remove the method. Thanks for the hint 👍